python/docs/hashtable_repeated_word/README.md

```diff
@@ -0,0 +1,41 @@
+# Challenge Summary
+Write a function called repeated word that finds the first word to occur more than once in a string
+  - Arguments: string
+  - Return: string
+
+
+## Whiteboard Process
+1[](hashmap-repeated-word.png)
+
+## Approach & Efficiency
+-  Write a function with argument string
+-  set a variable with empty set
+-  using regex filter out only the word characters
+-  lowercase all word chars and split them
+-  go through for loop for each word in the stripped string
+-  if word is in the set return the word
+-  if word is not present add the word to the set
+
+Big 0
+- Time: O(n): Since we are only iterating through the string until we find a matching word. However, if no matching word is - there then it is O(N).
+Space:O(1): Since we are only saving the first repeated word.
```

Comment:
It is right that the set will hold only the first repeated word. Though, consider this extreme case: how big is that set when the input string does not have any repeated words (all of the words in it are unique)? Can that set get bigger when the input string gets bigger?